sycl/source/detail/config.def

```diff
@@ -38,3 +38,6 @@ CONFIG(INTEL_ENABLE_OFFLOAD_ANNOTATIONS, 1, __SYCL_INTEL_ENABLE_OFFLOAD_ANNOTATI
 CONFIG(SYCL_ENABLE_DEFAULT_CONTEXTS, 1, __SYCL_ENABLE_DEFAULT_CONTEXTS)
 CONFIG(SYCL_QUEUE_THREAD_POOL_SIZE, 4, __SYCL_QUEUE_THREAD_POOL_SIZE)
 CONFIG(SYCL_RT_WARNING_LEVEL, 4, __SYCL_RT_WARNING_LEVEL)
+CONFIG(SYCL_REDUCTION_MAX_WORKGROUP_SIZE_CPU, 16, __SYCL_REDUCTION_MAX_WORKGROUP_SIZE_CPU)
+CONFIG(SYCL_REDUCTION_MAX_WORKGROUP_SIZE_GPU, 16, __SYCL_REDUCTION_MAX_WORKGROUP_SIZE_GPU)
+CONFIG(SYCL_REDUCTION_MAX_WORKGROUP_SIZE_ACC, 16, __SYCL_REDUCTION_MAX_WORKGROUP_SIZE_ACC)
```

Comment:
Should be 4 for 'positive integer'
```suggestion
CONFIG(SYCL_REDUCTION_MAX_WORKGROUP_SIZE_CPU, 4, __SYCL_REDUCTION_MAX_WORKGROUP_SIZE_CPU)
CONFIG(SYCL_REDUCTION_MAX_WORKGROUP_SIZE_GPU, 4, __SYCL_REDUCTION_MAX_WORKGROUP_SIZE_GPU)
CONFIG(SYCL_REDUCTION_MAX_WORKGROUP_SIZE_ACC, 4, __SYCL_REDUCTION_MAX_WORKGROUP_SIZE_ACC)
```

Reply:
The number seems to be the max number of characters consumable by the config. Since it uses `stoi` it produces an `int` it is constrained by it's max, which is 10 characters. We convert to `size_t`, so we could change to use `stoll` instead, which can take up to 19 characters, but for max work-group size I think max `int` should be more than enough. Should we go with 8, 10, or keep 16 you think? Some other positive integer configs, like `SYCL_CACHE_THRESHOLD`, use 16.

Reply:
I thought, those were bytes. Ok, 16 is good, IMO.